levels/house_ceremony/events.js

```diff
@@ -39,28 +40,48 @@ module.exports = async function (event, world) {
 
   // Once the player has selected their house, the fire is no longer interactable; Lovelace tower opens; 
   // the fires change color
-  if (worldState.playerHouse){
+  if (worldState.playerHouse)
+  {
     world.forEachEntities("fire-convo", fire => {
       fire.isConversationDisabled = true;
       fire.interactable = false;
     });
     world.showEntities(`${worldState.playerHouse}-fire`);
 
-    // [] Give avatar item based on chosen house
+    // Give avatar item based on chosen house
+    world.grantItems([`academy_jacket_${worldState.playerHouse}`, "academy_pleated_skirt", "academy_slacks"]);
     
-    // [] Pledge scroll fade animation is triggered
+    // When house-fire convo closes, pledge scroll fade animation is triggered - half second delay; should only play once and then be hidden again
+    if (
+      event.name === "conversationDidEnd" &&
+      event.npc.conversation === "house-fire"
+      ){
+        world.showEntities(`pledge-scroll`);
+        world.hideEntities(`pledge-scroll`, 750);
+        // world.forEachEntities(`pledge-scroll`, async scroll =>{
+        //   await scroll.playAnimation("idle");
+        //   scroll.hidden = true;
+        //   });
```

Comment:
Is this meant to be commented out? If so, let's go ahead and remove it to avoid clutter!